bb.edn

```diff
@@ -1,2 +1,2 @@
-{:deps     {local/deps {:local/root "."}}
+{:deps     {io.github.eval/deps-try {:local/root "."}}
```

Comment:
It's better to use a unique name to not cause conflicts with other local libs named `local/deps`.